pino.js

```diff
@@ -34,24 +34,40 @@ var levels = {
   debug: 20,
   trace: 10
 }
+Object.freeze(levels)
 
 var nums = Object.keys(levels).reduce(function (o, k) {
   o[levels[k]] = k
   return o
 }, {})
+Object.freeze(nums)
```

Comment:
Yes, my thoughts as well. However they can be removed as well, I just kept them to prevent future bugs like this from happening.